prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/UpdateCommand.java

```diff
@@ -129,14 +151,32 @@ private void performUpdate(UpdateAction updateAction) throws ArtifactResolutionE
             return;
         }
 
-        updateAction.performUpdate();
+        final List<FileConflict> fileConflicts = updateAction.performUpdate();
+
+        FileConflictPrinter.print(fileConflicts, console);
 
         console.updatesComplete();
     }
 
+    private void buildUpdate(UpdateAction updateAction) throws OperationException, ProvisioningException {
+        final UpdateSet updateSet = updateAction.findUpdates();
+
+        console.updatesFound(updateSet.getArtifactUpdates());
+        if (updateSet.isEmpty()) {
+            return;
+        }
+
+        if (!yes && !console.confirmBuildUpdates()) {
+            return;
+        }
+
+        updateAction.buildUpdate(updateDirectory.get().toAbsolutePath());
+
+        console.buildUpdatesComplete();
+    }
```

Comment:
Could we print conflicting files during `buildUpdate()` too? For user convenience...

Reply:
We could, but I'm not sure if it belongs at buildUpdate stage. There might be further changes before user does applyUpdate - would we want to display them twice?